go.mod

```diff
@@ -1,65 +1,74 @@
 module github.com/newrelic/nri-kubernetes/v3
 
-go 1.17
+go 1.18
 
 require (
 	github.com/google/go-cmp v0.5.7
 	github.com/newrelic/infra-integrations-sdk v3.7.2+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/prom2json v1.3.0
+	github.com/prometheus/prom2json v1.3.1
 	github.com/segmentio/go-camelcase v0.0.0-20160726192923-7085f1e3c734
 	github.com/sethgrid/pester v1.2.0
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/viper v1.10.1
-	github.com/stretchr/testify v1.7.0
-	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.22.3
-	k8s.io/apimachinery v0.22.3
-	k8s.io/client-go v0.22.3
-	k8s.io/kubelet v0.22.3
+	github.com/spf13/viper v1.11.0
+	github.com/stretchr/testify v1.7.1
+	google.golang.org/protobuf v1.28.0
+	k8s.io/api v0.23.6
+	k8s.io/apimachinery v0.23.6
+	k8s.io/client-go v0.23.6
+	k8s.io/kubelet v0.23.6
+)
+
+replace (
+	// To avoid CVE-2018-1098 triggering a security scan
+	go.etcd.io/etcd v3.3.27+incompatible => go.etcd.io/etcd/v3 v3.5.4
+	// This is a dependency of golint, trivy is mixing up dependencies
```

Comment:
I think it is a really-nested dependency from `k8s.io/apimachinery` too... maybe trivy is not mixing that up

Reply:
To be honest, I still not understand the whole go toolset so I would leave it as this until trivy raise another issue and I have more experience.